zerver/openapi/zulip.yaml

```diff
@@ -11807,7 +11807,9 @@ paths:
 
         Note that administrators can manage invitations that were created by other users.
 
-        **Changes**: Prior to Zulip 8.0 (feature level 209), non-admin users could
+        **Changes**: In Zulip 9.0 (feature level 247), added option for referrer to receive direct notifications from the notification bot when an invited user joins.
```

Comment:
The response for `GET /invites` does not include the `notify_referrer_on_join` field, so no need for these changes. Though might be good to confirm whether we want to send the data to clients so that the user can know whether they will get the notification or not.

Reply:
I don't think it's important to display at this time. It would be most consistent to expose in the API data, since we usually return all meaningful non-secret fields.